pyuavcan/transport/can/media/pythoncan/_pythoncan.py

```diff
@@ -174,22 +211,22 @@ def _read_frame(self) -> _media.TimestampedDataFrame:
                 out = self._parse_native_frame(msg, loopback=loopback, timestamp=timestamp)
                 if out is not None:
                     return out
-                
+
     @staticmethod
     def _parse_native_frame(msg: can.Message,
                             loopback: bool,
                             timestamp: pyuavcan.transport.Timestamp) \
             -> typing.Optional[_media.TimestampedDataFrame]:
         if (msg.error_state_indicator == True) or (msg.is_error_frame == True):  # error frame, ignore silently
```

Comment:
If I'm reading the PythonCAN docs correctly, [`error_state_indicator`](https://python-can.readthedocs.io/en/master/message.html#can.Message.error_state_indicator) merely indicates that the adapter is in the error active state, so it should not be checked here.

```suggestion
        if msg.is_error_frame:  # error frame, ignore silently
```

Reply:
Yes, you ar right